### PR TITLE
Rokita/clk1 cors rerun (3/4)

### PR DESCRIPTION
**Pull request by jharenza**
#### Reproducibility Checklist

- [ ] The dependencies required to run the code in this pull request have been added to the project Dockerfile.

#### Documentation Checklist

- [ ] This analysis module has a `README` and it is up to date.
- [ ] The analytical code is documented and contains comments.

### CODE REVIEW

**Review by naqvia — DISMISSED** [review]
Runs as expected but changed path to clk1 splice events, bc it speeds up exponentially No reason to look at all pbta!

**Comment by jharenza** [comment]
> Runs as expected but changed path to clk1 splice events, bc it speeds up exponentially No reason to look at all pbta!

true, but this is the module which was used to create that clk1 splice event file for data release, so it should remain.

File: analyses/CLK1-splicing_correlations/04-CLK1_PSI_plots.R [hunk 1]
```diff
@@ -43,7 +43,7 @@ figures_dir <- file.path(root_dir, "figures")
 source(file.path(figures_dir, "theme_for_plots.R"))
 
 ## input files
-rmats_file <- file.path(data_dir,"splice-events-rmats.tsv.gz")
+rmats_file <- file.path(data_dir,"clk1-splice-events-rmats.tsv")
```
Comment (jharenza) [inline analyses/CLK1-splicing_correlations/04-CLK1_PSI_plots.R:46]:
```suggestion
rmats_file <- file.path(results_dir, "clk1-splice-events-rmats.tsv")
```

**Comment by jharenza** [comment]
@naqvia updated paths, ready to re-review.

**Review by naqvia — APPROVED** [review]
runs as expected!

**Pull request merged**